cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -163,27 +180,48 @@ script "js_target_util", type:"javascript" do
   code "result = { value: (param_target_util / 100).toString() }"
 end
 
-datasource "ds_cluster_sizing" do
+datasource "ds_clusters" do
   request do
     host $param_kubecost_host
-    path "/model/savings/clusterSizing"
+    path "/model/savings/clusterSizingETL"
     query "minNodeCount", $param_min_nodes
     query "window", join([$param_lookback, "d"])
     query "targetUtilization", val($ds_target_util, "value")
+    query "allowSharedCore", $param_allow_shared_core
+    query "architecture", $param_architecture
+    query "includeOverhead", "false"
```

Comment:
let's keep a note here adding why the `includeOverhead` field is included and set to `false`

Reply:
done!